src/components/cars-components/cars.js

```diff
@@ -0,0 +1,68 @@
+const items = [
```

Comment:
- [ ] As you pointed out in your PR description, the main cars list page is static.  When I add a car, it will appear in the "Delete" section, and "Booking" section, but not in the main Cars section.

I understand this is a work in progress.  Keep it up! 💪